Lib/importlib/_bootstrap_external.py

```diff
@@ -1635,3 +1635,29 @@ def _install(_bootstrap_module):
     supported_loaders = _get_supported_file_loaders()
     sys.path_hooks.extend([FileFinder.path_hook(*supported_loaders)])
     sys.meta_path.append(PathFinder)
+
+
+def alternative_architectures(suffixes):
```

Comment:
We should probably make this explicitly internal

```suggestion
def _alternative_architectures(suffixes):
```